lib/osf-components/addon/components/form-controls/radio-button-group/component.ts

```diff
@@ -1,38 +1,35 @@
 import { tagName } from '@ember-decorators/component';
 import Component from '@ember/component';
 import { assert } from '@ember/debug';
-import { action } from '@ember/object';
 import { BufferedChangeset } from 'ember-changeset/types';
 
 import { layout } from 'ember-osf-web/decorators/component';
+import { SchemaBlock } from 'ember-osf-web/packages/registration-schema';
 
 import styles from './styles';
 import template from './template';
 
+export interface RadioButtonOption {
+    displayText: string;
+    inputValue: string | boolean | number;
+}
+
 @tagName('')
 @layout(template, styles)
-export default class FormControlRadioButton extends Component {
+export default class FormControlRadioButtonGroup extends Component {
     // Required params
-    options!: string[];
+    options!: string[] | SchemaBlock[];
     valuePath!: string;
     changeset!: BufferedChangeset;
 
     // Optional params
     shouldShowMessages?: boolean;
     disabled = false;
-    onchange?: (option: string) => void;
+    onchange?: (option: string | number | boolean) => void;
```

Comment:
This was missing before this PR, but it looks like this component should take an optional param called `helpTextMapping: any`. Can we add that here?